### ISSUE TITLE
In IE8 all responses fails when XMLHTTP option disabled.

### ISSUE THREAD

**Issue by Aradesh** [doedje/jquery.soap#72]
Hi,
the problem is in soapEnvelope.send() method, where is used xhrFields object

```javascript
xhrFields: {
  onprogress: function(e) {
     if (e.lengthComputable) {
	 log("jquery.soap - progress:", (e.loaded / e.total * 100));
     } else {
	 log("jquery.soap - progress:","Length not computable.");
     }
   }
 }
```
More info about this bug: 
http://bugs.jquery.com/ticket/12750

Without this object IE8 worked correctly, even XMLHTTP option is disabled in Settings. Maybe the problem is in ActiveXObject('MSXML2.XMLHTTP.3.0') that emulate XMLHttpRequest
https://msdn.microsoft.com/en-us/library/ie/ms537505%28v=vs.85%29.aspx that not supports xhrFields

**Comment by Aradesh** [comment]
UPD: Same with xhr in new versions of plugin

**Comment by doedje** [comment]
Thanx for pointing out.. I will disable the piece of code since it does not do much at the moment anyway....

**Comment by doedje** [comment]
I don't have access to IE8, so would you test the following code for me in the 1.6.0 version of $.soap?

I wrapped the code in a `if (window.XMLHttpRequest)` which should make IE8 skip the whole thing...
```
xhr: function() {
	if (window.XMLHttpRequest) {
		var xhr = new window.XMLHttpRequest();
		xhr.upload.addEventListener("progress", function(evt) {
			if (evt.lengthComputable) {
				var progress = evt.loaded / evt.total;
				log("jquery.soap - progress up: ", (progress * 100) + '% total: ' + evt.total);
			}
		}, false);
		xhr.addEventListener("progress", function(evt) {
			if (evt.lengthComputable) {
				var progress = evt.loaded / evt.total;
				log("jquery.soap - progress down: ", (progress * 100) + '% total: ' + evt.total);
			}
		}, false);
		return xhr;
	}
}
```

Thanx, hope to hear from you!

**Comment by Aradesh** [comment]
I will do it today.
Actualy xhr callback breaks the requests even if this function is empty.
I can fix this and then create pull request if you need.

**Comment by doedje** [comment]
Hi Aradesh,

I was wondering if the code above is still breaking IE8.... When you have the time to make a fix and send me a pull request I'd be very thankful! =]

**Comment by doedje** [comment]
I think this issue is resolved with the 1.6.6 version I just released 5 minutes ago...
